spec/contact_spec.rb

```diff
@@ -1,34 +1,34 @@
 require 'contact'
 
 RSpec.describe Contact do
-  it "creates a new instance of the Contact class" do
+  xit "creates a new instance of the Contact class" do
     my_contacts = Contact.new
     expect(my_contacts.is_a?Object).to eq(true)
   end
 
-  it "creates a new contact using the Person class and adds it to @contacts" do
+  xit "creates a new contact using the Person class and adds it to @contacts" do
     allow($stdin).to receive(:gets).and_return("Elle", "H", "elle@hello.com", "07999999999")
     my_contacts = Contact.new
     my_contacts.create_new_entry
     expect(my_contacts.contacts.empty?).to eq(false)
   end
 
-  it "create_new_entry method accepts input, creates new instance of Person and adds to @contacts" do
+  xit "create_new_entry method accepts input, creates new instance of Person and adds to @contacts" do
     allow($stdin).to receive(:gets).and_return("Elle", "H", "elle@hello.com", "07999999999")
     contacts = Contact.new
     contacts.create_new_entry
     expect(contacts.contacts).to eq([{"first_name" => "Elle", "last_name" => "H", "email_address" => "elle@hello.com", "phone_number" => "07999999999"}])
   end
 
-  it "create_new_entry method accepts input, creates new instance of Person and adds to @contacts twice" do
+  xit "create_new_entry method accepts input, creates new instance of Person and adds to @contacts twice" do
     allow($stdin).to receive(:gets).and_return("Elle", "H", "elle@hello.com", "07999999999", "Deneice", "Daniels", "dee@hello.com", "07889999999")
```

Comment:
There's a way you can reuse this logic for each of your tests and not have to repeat yourself, with a `before` block. Before your first test you can put something like:

```ruby
before(:each) do
   allow($stdin).to receive(:gets).and_return("Elle", "H", "elle@hello.com", "07999999999")
end
```

Reply:
Thank you 👍

Reply:
For some reason, this doesn't seem to work when I call a method multiple times, like this:

`my_contacts = Contact.new
    my_contacts.create_new_entry
    my_contacts.create_new_entry
    my_contacts.create_new_entry
    expect(my_contacts.search_phone)...`

I can't seem to figure out why. Do you have any idea?